### ISSUE TITLE
Orchis theme installed in libadwaita does not follow Light/Dark switching

### ISSUE THREAD

**Issue by C4NoN424** [vinceliuice/Orchis-theme#243]
Hi.
The libadwaita Orchis theme installed by the command `./install.sh -l` does not change even if I switch between Light and Dark in the appearance settings of gnome-control-center. 
I know that by adding `-c [light|dark]` to the above command and specifying a color, I can reflect that color theme, but for example, an extension can schedule a switch between Light and Dark. When doing so, it is troublesome to enter commands one by one, and it is inconvenient that the changed libadwaita is not reflected unless the application is reopened.

I'm using GNOME42 on Arch Linux.

I want this problem fixed because I want to use GNOME conveniently with the look I like.

**Comment by vinceliuice** [comment]
Run: `./install.sh -r -l` to remove links for libadwaita

**Comment by vinceliuice** [comment]
Gnome team make libadwaita not support custom themes now, so if you link this theme for gtk4 config, then it'll not support switch between Light and Dark

**Comment by scratchmex** [comment]
Is there an issue we can follow about this in the libadwaita repo or something?

**Comment by vinceliuice** [comment]
Nope, Gnome team not gona support custom themes any more, if you are using arch based linux system you can use: https://aur.archlinux.org/packages/libadwaita-without-adwaita-git to make it support custom themes, or you can just use gtk-4.0 config file to make all gtk4.0 app use one custom theme

**Comment by MalonnBethesda** [comment]
I'm on a Fedora based distro with GNOME 42 and have had the same problem as OP.  You said use a gtk-4.0 config file?  Where is it?  Where should it be?  Is there some documentation in finding/creating the file?  I like your theme and would like to use is system wide.  I can't find shit online so far.

**Comment by MalonnBethesda** [comment]
Okay, I guess you can find "shit" online.  I was able to set the GTK theme globally by adding the line "GTK_THEME=<THEME>" to my /etc/environment file.

**Comment by vinceliuice** [comment]
@MalonnBethesda Only default libadwaita supoort dark mode

**Comment by MalonnBethesda** [comment]
Oh, okay.  Well, when using the `install.sh -l` option, certain dialogs like the settings window and calculator were not following my theme.  But adding `GTK_THEME=THEME` to the /etc/environment file fixed it.  All is good.  I must be using the default libadwaita library?

**Comment by C4NoN424** [comment]
> Nope, Gnome team not gona support custom themes any more, if you are using arch based linux system you can use: https://aur.archlinux.org/packages/libadwaita-without-adwaita-git to make it support custom themes, or you can just use gtk-4.0 config file to make all gtk4.0 app use one custom theme

I have tried 'libadwaita-without-adwaita'. However, it seems to reference adwaita every time I switch between light and dark, and when I press the theme switch button, the appearance is broken because a non-existent adwaita is applied. I was using the 'User theme X' extension to specify a GTK theme for light/dark switching, but found that as soon as I hit the light/dark switch button, it references an unspecified GTK theme and rewrites the settings. I could not tell if this was a GNOME specification or a bug in User theme X.

To fix this, you need to install the 'Night Theme Switcher' extension, turn off auto scheduling and set the GTK theme you want to use when dark/light theme. However, it seems that this extension is designed not to remember the state of the dark/light theme before logging out when logging in again. 

I also found that applying the orchis theme with 'libadwaita without adwaita' and doing light/dark switching when using the 'night them switcher' makes it difficult to see the title bar black while the font is black for applications using libadwaita.
